doc/OnlineDocs/library_reference/kernel/index.rst

```diff
@@ -91,6 +91,96 @@ A simplified version of this using :python:`pyomo.environ` components might look
 
 The transformer expressed using :python:`pyomo.kernel` components requires roughly 2 KB of memory, whereas the :python:`pyomo.environ` version requires roughly 8.4 KB of memory (an increase of more than 4x). Additionally, the :python:`pyomo.kernel` transformer is fully compatible with all existing :python:`pyomo.kernel` block containers.
 
+Direct Support For Conic Constraints with Mosek
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pyomo 5.6.2 introduced support into :python:`pyomo.kernel`
```

Comment:
I think this will end up being Pyomo 5.6.3 unless you instead want to reference the first version of Pyomo to include the Mosek interface.

Reply:
Thanks. I meant to ask that.